BUILD

```diff
@@ -1 +1 @@
-python_requirements(requirements_relpath="test_docker_requirements.txt")
\ No newline at end of file
+python_requirements(source="test_docker_requirements.txt")
```

Comment:
The name of this keyword argument has changed: https://www.pantsbuild.org/docs/python-third-party-dependencies#requirementstxt